docker/base-python/Dockerfile

```diff
@@ -24,12 +24,32 @@ WORKDIR /buildroot
 
 ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/buildroot/bin
 
+# For the most-part, you should not pin specific versions in this
+# `apk` command; it will always choose the latest version being
+# shipped by pinned version of Alpine.  This will automatically pull
+# in any patches, but not breaking changes.
+#
+# Because of how we cache the base image
+# (`docker/base-python.docker.gen`), it'll keep cache that for a week.
+# If there's a security patch patch that we want to pull in without
```

Comment:
extra word patch.

Also `Because of how we cache the base image (docker/base-python.docker.gen), it'll keep cache that for a week` needs a grammar check